src/main/cli/settings.c

```diff
@@ -1637,19 +1637,19 @@ const clivalue_t valueTable[] = {
 #ifdef I2C_FULL_RECONFIGURABILITY
 #ifdef USE_I2C_DEVICE_1
     { "i2c1_pullup",    VAR_UINT8  | HARDWARE_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_OFF_ON }, PG_I2C_CONFIG, PG_ARRAY_ELEMENT_OFFSET(i2cConfig_t, 0, pullUp) },
-    { "i2c1_overclock", VAR_UINT8  | HARDWARE_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_OFF_ON }, PG_I2C_CONFIG, PG_ARRAY_ELEMENT_OFFSET(i2cConfig_t, 0, overClock) },
+    { "i2c1_clockspeed", VAR_UINT16 | HARDWARE_VALUE, .config.minmax = { I2C_CLOCKSPEED_MIN, I2C_CLOCKSPEED_MAX }, PG_I2C_CONFIG, PG_ARRAY_ELEMENT_OFFSET(i2cConfig_t, 0, clockSpeed) },
```

Comment:
These will become a lot more self-explanatory if they indicate the units as `i2c<x>_clockspeed_khz`.

And the same goes for `I2C_CLOCKSPEED_MIN_KHZ` and `I2C_CLOCKSPEED_MAX_KHZ`.

Reply:
Done!